cmd/fetchd/cmd/stakereconciliationmigrate.go

```diff
@@ -273,17 +270,19 @@ Eligible accounts:
 
 			fmt.Println(string(sortedBz))
 
-			for _, _ineligible := range ineligible {
-				_ineligible.Print()
+			if flagDebug {
+				for _, _ineligible := range ineligible {
+					_ineligible.Print()
+				}
 			}
 
 			return nil
 		},
 	}
-	cmd.Flags().StringVarP(&stakesCSVPath, "stakes-csv", "s", "./staked_export.csv", `Path to a CSV file containing legacy stake-holder addresses; fields: ETH_ADDR,PUB_KEY,ORIG_FETCH_ADDR,MIGRATE_AMOUNT`)
-	cmd.Flags().StringVarP(&registrationsPath, "registrations", "r", "./registrations.json", "Path to a JSON file containing the result of querying reconciliation contract registrations")
-	cmd.Flags().StringVarP(&outputPath, "output", "o", "./genesis_out.json", "path to write migrated genesis JSON data")
-	cmd.Flags().StringVarP(&coinDenom, "coin-denom", "c", "afet", "coin denomination to use when checking eligibility and migrating balances")
+	cmd.Flags().StringVarP(&flagStakesCSVPath, "stakes-csv", "s", "./staked_export.csv", `Path to a CSV file containing legacy stake-holder addresses; fields: ETH_ADDR,PUB_KEY,ORIG_FETCH_ADDR,MIGRATE_AMOUNT`)
+	cmd.Flags().StringVarP(&flagRegistrationsPath, "registrations", "r", "./registrations.json", "Path to a JSON file containing the result of querying reconciliation contract registrations")
+	cmd.Flags().StringVarP(&flagCoinDenom, "coin-denom", "c", "afet", "coin denomination to use when checking eligibility and migrating balances")
+	cmd.Flags().BoolVarP(&flagDebug, "debug", "d", false, "prints ineligible old addresses to stderr")
```

Comment:
Not sure what happened here, is it to match flag usage of the other migration command?

Then it would need to be:
```
cmd.Flags().Bool(flagDebug, false, "prints ineligible old addresses to stderr")
```
with a definition of the flag name on top of the file:
```
const flagDebug = "debug"
``` 
and add the lines to read the flag value somewhere in the command itself.

But you can keep using the `flag.StringVar` form, and the previous names were all fine (flagXXX constant being mostly used for defining flag names in cosmos)

Reply:
Gotcha, I was going for consistency with the names but didn't realize there's a grander convention at work. I'm change them back for now.